client/src/main/proto/spine/base/command.proto

```diff
@@ -95,7 +95,7 @@ message CommandContext {
 // Enumeration of possible failures when validating a command.
 enum CommandValidationError {
     // Reserved value.
-    UNKNOWN = 0;
+    UNDEFINED_COMMAND_VALIDATION_ERROR = 0;
```

Comment:
Why do we need this long constant name assuming the enum is already named `CommandValidationError`?

Why do we want the user to read `CommandValidationError.UNDEFINED_COMMAND_VALIDATION_ERROR`, instead of  `CommandValidationError.UNDEFINED`?